packages/arb-token-bridge-ui/src/state/app/utils.ts

```diff
@@ -307,15 +305,34 @@ export const isDepositReadyToRedeem = (tx: MergedTransaction) => {
   return isDeposit(tx) && tx.depositStatus === DepositStatus.L2_FAILURE
 }
 
-export const getStandardizedTimestamp = (date: string | BigNumber) => {
+export const normalizeTimestamp = (date: number | string | BigNumber) => {
```

Comment:
need to use this at `redeemRetryable.cy.ts` L29

Reply:
updated